scripts/run-circle.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+if [ "$CIRCLE_BRANCH" != "master" ] && [ "$CI_PULL_REQUESTS" == "" ] ; then
```

Comment:
Working out what this condition is trying to do, and why it’s there, is causing my (admittedly quite tired) brain to trip over itself.

Worth a comment?